ArticleTemplates/liveblogTemplate.html

```diff
@@ -149,6 +149,8 @@ <h1 class="the-minute__headline selectable">__TITLE__</h1>
             <img src="__TEMPLATES_DIRECTORY__assets/img/loading.gif" alt="Loading…"/>
         </div>
 
+        __CONTRIBUTIONS_EPIC__
```

Comment:
This __CONTRIBUTIONS_EPIC__ placeholder should be moved out of this template and into liveBlogTemplateBlock.html beneath the div.block element. Keeping it within this template would mean it's always at the bottom of a liveblog, moving it to the block template means it will appear between liveblog blocks.